docs/topics/pagination.txt

```diff
@@ -81,72 +81,71 @@ accessing the items for each page::
 Paginating a ``ListView``
 =========================
 
-:class:`django.views.generic.list.ListView` provides a builtin way to paginate
-the displayed list. You can do this by adding
+:class:`django.views.generic.list.ListView` provides a built in way to paginate
```

Comment:
I think that `built-in` is the form used in other points of the documentation.

Reply:
Yes, fair enough.  Thanks for looking.  [This is my first pull request so I am treading carefully.  I think I can make that change and commit it.  I'll do that later today unless I hear otherwise.]